.fixtures.yml

```diff
@@ -24,6 +24,7 @@ fixtures:
       puppet_version: '>= 6.0.0'
     stdlib:        "https://github.com/puppetlabs/puppetlabs-stdlib.git"
     systemd:       "https://github.com/camptocamp/puppet-systemd.git"
+    transition:    "https://github.com/puppetlabs/puppetlabs-transition.git"
```

Comment:
Nit: I'd do this as a separate commit. Fine to do it in the same PR.

Reply:
Split out